### DISCUSSION TITLE
Dynamic Phantom

### DISCUSSION THREAD

**Discussion by FasicolZyy** [e0404/matRad#620] [Q&A] [answered]
Hi
        the tumor inside my matrad phantom is currently static. I would like to know which parameters I should adjust to make the tumor move with the time period of the human body’s breathing.

**Comment by wahln** [comment]
The dev_varRBErobOpt branch has a few functionalities for 4D optimization and dose calculation. It is not thoroughly tested and documented yet, but you may take a look at examples 9 and 10 there. These examples create moving phantoms and do robust optimization (in case of example 10).

**Reply by FasicolZyy** [reply, marked as answer]
Thank you very much,this really helps me a lot
